jsettlers.graphics.androidui/src/main/java/jsettlers/graphics/androidui/menu/AndroidMenu.java

```diff
@@ -32,7 +32,7 @@ public AndroidMenu(AndroidMenuPutable puttable) {
 		this.putable = puttable;
 	}
 
-	protected Context getContext() {
+	public Context getContext() {
```

Comment:
probably just briefly from my early work on this. I don't think my version uses this class anymore